AdyenCard/Utilities/BinLookup/BinLookupResponse.swift

```diff
@@ -17,7 +17,7 @@ internal struct BinLookupResponse: Response {
     internal let requestId: String
 
     /// :nodoc:
-    internal let issuingCountryCode: String
+    internal let issuingCountryCode: String?
```

Comment:
is there a case where this can be `nil`? when decoding a JSON?

Reply:
Yes, when decoding a JSON

In normal condition:

```
{
  "issuingCountryCode" : "US",
  "brands" : [
    {
      "cvcPolicy" : "required",
      "brand" : "visa",
      "enableLuhnCheck" : true,
      "showExpiryDate" : true,
      "showSocialSecurityNumber" : false,
      "supported" : true
    }
  ],
  "requestId" : "AF41AC8D-1F7E-48D1-98B3-4DABED429F2D"
}
```


If BIN is unknown the result is :

```
{
  "requestId" : "BA050BE8-3AE0-4D87-9FBA-6DC4E2C382B4"
}
```